frontend/src/scenes/insights/views/WorldMap/countryCodes.ts

```diff
@@ -510,7 +517,13 @@ export const languageCodeToName: Record<string, string> = {
 
 // This is only used as a fallback for some languages that don't usually
 // come in the locale-country format (such as nl-NL usually being presented simply as nl)
-// We'll fill this as we see fit based on the values seen in the wild
+// but that can't simply be translated to a country by capitalizing the locale
 const languageCodeToCountryCode: Record<string, string> = {
-    nl: 'NL',
+    ar: 'SA', // Arabic -> Saudi Arabia, tricky, there's no good representation for the "default Arab country"
+    af: 'ZA', // Afrikaans -> South Africa
+    ta: 'IN', // Tamil -> India
+    eu: 'ES', // Basque -> Spain, tricky, but there's no better flag than the Spanish one, we could move to custom SVGs if we wanted to solve this
+    cy: 'GB', // Welsh -> UK
+    ne: 'NP', // Nepali -> Nepal
+    zz: '', // Unknown Language -> No Emoji
```

Comment:
🏴󠁥󠁳󠁰󠁶󠁿 might be a better choice for unknown, so that text is still aligned

Reply:
What flag were you trying to render here? I see a black flag, and if I try to copy/paste it I see there's a`ESPVF` code after that, but I don't recognize that.

I've pushed a change with the UN flag. What's your opinion on that?

Reply:
Oh weird, it was meant to just be the black flag